RFS/src/test/java/org/opensearch/migrations/bulkload/integration/SnapshotStateTest.java

```diff
@@ -95,7 +95,7 @@ public void SingleSnapshot_SingleDocument() throws Exception {
         verify(client, times(1)).sendBulkRequest(eq(indexName), docsCaptor.capture(), any());
         final var document = docsCaptor.getValue().get(0);
         assertThat(document.getDocId(), equalTo(document1Id));
-        assertThat(document.asBulkIndex(), allOf(containsString(document1Id), containsString("{\"fo$o\":\"bar\"}")));
+        assertThat(document.asString(), allOf(containsString(document1Id), containsString("{\"fo$o\":\"bar\"}")));
```

Comment:
what's the purpose of this line?  asString() sounds pretty generic and I'd be surprised if the contract required me to return anything in particular?  It would have been much less surprising when the method was called 'asBulkIndex'

Reply:
I'll rename to asBulkIndexString